pygmt/io.py

```diff
@@ -0,0 +1,36 @@
+"""
+PyGMT input/output (I/O) utilities.
+"""
+import xarray as xr
+
+
+def process_output_grid(grid_name, tmpfile_name):
+    """
+    Processes the output from the GMT API to return an xarray.DataArray if
+    ``grid_name`` matches ``tmpfile_name`` and return None if it does not.
+
+    Parameters
+    ----------
+    grid_name : str
+        The name of the output netCDF file with extension .nc to store the grid
+        in.
+    tmpfile_name : str
+        The name attribute from a GMTTempFile instance.
```

Comment:
Copied directly from the `filename_or_obj` parameter docs in [`xarray.open_dataarray`](https://xarray.pydata.org/en/v0.19.0/generated/xarray.open_dataarray.html#xarray.open_dataarray) at https://github.com/pydata/xarray/blob/a21dead8621480b608827355b50e85af6f88f185/xarray/backends/api.py#L542-L547

```suggestion
    filename_or_obj : str or Path or file-like or DataStore
        Strings and Path objects are interpreted as a path to a netCDF file
        or an OpenDAP URL and opened with python-netCDF4, unless the filename
        ends with .gz, in which case the file is gunzipped and opened with
        scipy.io.netcdf (only netCDF3 supported). Byte-strings or file-like
        objects are opened by scipy.io.netcdf (netCDF3) or h5py (netCDF4/HDF).
```